coderd/database/db2sdk/db2sdk.go

```diff
@@ -694,3 +695,13 @@ func MatchedProvisioners(provisionerDaemons []database.ProvisionerDaemon, now ti
 	}
 	return matched
 }
+
+func TemplateRoleActions(role codersdk.TemplateRole) []policy.Action {
+	switch role {
+	case codersdk.TemplateRoleAdmin:
+		return []policy.Action{policy.WildcardSymbol}
+	case codersdk.TemplateRoleUse:
+		return []policy.Action{policy.ActionRead, policy.ActionUse}
+	}
+	return []policy.Action{}
+}
```

Comment:
I wanted to place this in `codersdk`, but then I'd have to import `policy`. And we already have `codersdk.RBACAction`. 

`TemplateRole` should probably be a database enum. At present it only exists in `codersdk.`